Makefile

```diff
@@ -1,3 +1,13 @@
+default: help
+
+.PHONY: help
+help: ## Print this help message
+	@echo "Available make commands:"; grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
```

Comment:
Boilerplate I often add to Makefiles to print a nice help message. 
<img width="497" alt="Screen Shot 2022-04-21 at 4 48 56 PM" src="https://user-images.githubusercontent.com/224251/164563993-04cb32ef-c1c3-4c9c-8330-1157e8b27354.png">